release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -481,17 +481,12 @@ See xref:../networking/routes/route-configuration.adoc#nw-ingress-creating-a-rou
 [id="ocp-4-11-osdk"]
 === Operator development
 
-[id="ocp-4-11-PAO-to-NTO"]
-==== Performance Addon Operator functions moved to the Node Tuning Operator
+[id="ocp-4-11-java-osdk"]
+==== Java Operator SDK (Technology Preview)
 
-In earlier versions of {product-title}, the Performance Addon Operator provided automatic, low latency performance tuning for applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator and all related artifacts on startup.
+Starting in {product-title} {product-version} as a Technology Preview feature, the Operator SDK includes the tools and libraries to develop a Java-based Operator. Operator developers can take advantage of Java programming language support in the Operator SDK to build a Java-based Operator and manage its lifecycle.
```

Comment:
I guess instead of Java we might need to mention it to Quarkus.

@jmrodri - WDYT?

Reply:
As discussed on slack, we shouldn't call it "Java Operator SDK" that is the upstream project name. It also gives the impression we have a separate SDK for Java. I think a better way is Quarkus based Java Operators to be exact.

Reply:
Java-based Operators is also acceptable since I can see us adding plain java to the plugin in the future.

Reply:
I agreed with Java-based Operators to keep it more generic for future purposes. Pushing the changes now!